### PR TITLE
README: Added `openssl` to OSX dependencies

### PR DESCRIPTION
**Pull request by elitan**
Could not run `make otherwise.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ postgres=# select * from t;
 
 ```sh
 git clone --recursive https://github.com/neondatabase/neon.git
-make # builds also postgres and installs it to ./tmp_install
+make # builds also postgres and installs it to ./tmp_insftall
```
Comment (SomeoneToIgnore) [inline README.md:194]: Do you mind reverting that? 🙂

**Review by SomeoneToIgnore — COMMENTED** [review]
Thank you for spotting.

File: README.md (hunk 1 above)
Comment (elitan) [reply README.md]: Oups, sorry for that!

Comment (SomeoneToIgnore) [reply README.md]: No worries and thanks again.

**Review by SomeoneToIgnore — APPROVED** [review]

**Pull request merged**